google-cloud-bigquery/src/main/resources/META-INF/native-image/com.google.cloud/google-cloud-bigquery/reflect-config.json

```diff
@@ -29,9 +29,62 @@
     "name":"java.nio.DirectByteBuffer",
     "methods":[{"name":"<init>","parameterTypes":["long","int"] }]
   },
+
+  {
+    "name":"org.apache.arrow.memory.BaseAllocator",
```

Comment:
I suppose these new reflections are needed because of arrow changes?

Reply:
Yes, last release had a major version upgrade of this dependency. See https://github.com/googleapis/sdk-platform-java/pull/3863